src/tactic/transform_decl.lean

```diff
@@ -22,7 +41,7 @@ do
     is_protected ← is_protected_decl src,
     let decl := decl.update_with_fun (name.map_prefix f) tgt,
     if is_protected then add_protected_decl decl else add_decl decl,
-    attrs.mmap' (λ n, copy_attribute n src tgt)
+    attrs.mmap' (λ n, try $ copy_attribute' n src tgt)
```

Comment:
Don't we want an error message when this fails?

Reply:
I don't think so? At least the way its set up now copy_attribute' fails if the src doesn't have the attribute, so we don't want this to fail if the first attribute in the list fails, instead it should carry on and try to copy the others.

Reply:
Oh... We definitely want `to_additive` to succeed if the original declaration doesn't have a certain attribute. 
However, we want it to fail if the original declaration has the attribute, but copying the attribute fails.
Maybe we want a `copy_attribute_if_the_original_declaration_has_it` tactic (with a shorter name) that succeeds if the original declaration doesn't have the specified attribute, but fails if it has, but copying the attribute fails.